### PR TITLE
Refinement

### PR DESCRIPTION
**Pull request by siyuan0322**
- Add helper message for mac users
- Clarify install prefix and dependency prefix
- Bump up vineyard version and arrow version
- CI improvement
- Add stubs for build and build-images

### CODE REVIEW

File: src/bashly.yml [hunk 1]
```diff
@@ -100,21 +107,19 @@ commands:
   - long: --v6d-version
     arg: v6d-version
     help: v6d version to clone
-    default: v0.11.3
+    default: v0.11.6
```
Comment (yecol) [inline src/bashly.yml]: is that possible for sth. like `latest` as default?

Comment (siyuan0322) [reply src/bashly.yml]: Could use main as latest.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -21,10 +21,14 @@ jobs:
...
 
-    - name: install bash.ly and build
+    - name: Install bash.ly and build
       run: |
         gem install bashly
         bashly generate
+
+    - name: Insert friendly prompt
+      run: |
+        sed -i '8i \  \printf "If you are using MacOS, run the following command to get a newer bash: \\n  brew install bash && bash\\n" >&2' ./gs
```
Comment (yecol) [inline .github/workflows/build.yml]: Please do not do this! 
See PR #10 , line37

Comment (siyuan0322) [reply .github/workflows/build.yml]: OK~

File: src/bashly.yml [hunk 3]
```diff
@@ -28,7 +35,7 @@ commands:
   - name: image
     help: Images to build.
     # Whitelist + required
-    allowed: [graphscope-dev-base, graphscope-dev, coordinator, analytical, interactive-frontend, interactive-executor, learning]
+    allowed: [graphscope-dev-base, graphscope-dev, coordinator, analytical, analytical-java, interactive, interactive-frontend, interactive-executor, learning, vineyard-dev, vineyard-runtime]
```
Comment (yecol) [inline src/bashly.yml:38]: provide a `all`?

**Pull request merged**